app/iht/utils/CommonHelper.scala

```diff
@@ -68,6 +68,10 @@ object CommonHelper {
     fmt.format(n)
   }
 
+  def stripLineBreaks(inputCopy: String) = {
```

Comment:
As this is only used in tests it would be better to put it in one of the test libraries, e.g. testhelpers.ContentChecker.